pkg/apis/acid.zalan.do/v1/util.go

```diff
@@ -83,6 +84,7 @@ func validateCloneClusterDescription(clone *CloneDescription) error {
 	return nil
 }
 
+// Success of the last operation on a Postgres cluster
```

Comment:
Why the last operation? Isn't it just for the current status, that was passed as an argument?